reference/curl/functions/curl-share-close.xml

```diff
@@ -0,0 +1,125 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- $Revision$ -->
+<refentry xml:id="function.curl-share-close" xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink">
+ <refnamediv>
+  <refname>curl_share_close</refname>
+  <refpurpose>Schließt einen cURL-Share-Handle</refpurpose>
+ </refnamediv>
+
+ <refsect1 role="description">
+  &reftitle.description;
+  <methodsynopsis>
+   <type>void</type><methodname>curl_share_close</methodname>
+   <methodparam><type>CurlShareHandle</type><parameter>share_handle</parameter></methodparam>
+  </methodsynopsis>
+  <para>
+   Schließt einen cURL-Share-Handle und gibt die Resourcen wieder frei.
```

Comment:
I am unsure about the translation of `cURL-Share`. 

Is `geteilter cURL-Handle` better? Personally for me, this sound wired.

Reply:
I'm also always unsure about this (as with multi vs mehrfach). I translated shared memory as "gemeinsamer Speicher" (?), but here I would probably use "shared".